basic_features/basic_save_game_info.py

```diff
@@ -33,70 +51,133 @@ def allFiles(self) -> list[str]:
 
 
 class BasicGameSaveGameInfoWidget(mobase.ISaveGameInfoWidget):
-    def __init__(self, parent: QWidget, get_preview: Callable[[Path], Path | None]):
+    _preview_width = 320
+
+    def __init__(
+        self,
+        parent: QWidget,
+        get_preview: Callable[[Path], Path | None] | None = None,
```

Comment:
You have a double `None` check which is redundant, you can simply default to `lambda p: None` and remove the possibility of having `None`.

This also applies to `get_preview`.

Reply:
Replaced the double `None` check and added all supported types to `get_preview`
https://github.com/ModOrganizer2/modorganizer-basic_games/blob/dd95e5a6aee4ffe27d1c6ac98ff03105f68fb24d/basic_features/basic_save_game_info.py#L59-L64